UI/FleetWnd.cpp

```diff
@@ -1842,7 +1756,20 @@ void FleetDataPanel::Init() {
             m_stat_icons.push_back({std::get<0>(entry), icon});
             icon->SetBrowseModeTime(tooltip_delay);
             icon->SetBrowseText(UserString(std::get<2>(entry)));
-            icon->InstallEventFilter(shared_from_this());
+            icon->RightClickedSignal.connect([entry](const GG::Pt& pt){
+                MeterType meter_type = std::get<0>(entry);
+
+                std::string meter_string = boost::lexical_cast<std::string>(meter_type);
+
+                auto zoom_article_action = [&meter_string]() { ClientUI::GetClientUI()->ZoomToMeterTypeArticle(meter_string);};
```

Comment:
same for this... double popups

Reply:
Confirmed.

Reply:
Fixed via 4574e50ccff2134536e036a3478bec08c7cfd80e